inference-engine/thirdparty/clDNN/tests/test_cases/reduce_gpu_test.cpp

```diff
@@ -75,7 +75,7 @@ struct reduce_accumulator {
         return acc;
     };
 
-    AccT accumulate(AccT& acc, AccT& input_val, cldnn::reduce_mode reduce_mode) {
+    AccT accumulate(AccT& acc, AccT& input_val, cldnn::reduce_mode reduce_mode, bool force_sum) {
```

Comment:
It's up to you, but: What about `sum_only` instead of `force_sum`? It will be more clear.

Reply:
updated it.